frontend/src/metabase/dashboard/components/DashCard/DashCardTabMenu/DashCardTabMenu.tsx

```diff
@@ -23,6 +27,22 @@ export function DashCardTabMenu({ dashCardId }: DashCardTabMenuProps) {
     return tabs.filter(t => t.id !== selectedTabId);
   }, [selectedTabId, tabs]);
 
+  const moveDashcard = useCallback(
+    (destTabId: number) => {
+      dispatch(moveDashCardToTab({ dashCardId, destTabId }));
```

Comment:
I'm not a fan of abbreviations this removes clarity from the code. Especially when its counterpart isn't abbreviated.

`sourceTabId` vs `destTabId` can we make this `destinationTabId`?